src/pipeline_edsl/pipeline.ml

```diff
@@ -899,6 +899,17 @@ module Compiler = struct
         let work_dir = work_dir // (to_file_prefix pipeline) ^ "_work_dir" in
         Seq2HLA.hla_type
           ~work_dir ~run_with:machine ~run_name:info.fragment_id ~r1 ~r2
+      | Bam_to_fastq(info_opt, how, what) -> 
+        let fastq_pair = fastq_sample_step ~compiler sample in
```

Comment:
Ah yea, I see what you mean. I had thought that it would not give me `fragment_id` as needed above, but I see that it does